rust/signed_doc/src/validator/mod.rs

```diff
@@ -47,18 +60,20 @@ fn document_rules_init() -> HashMap<Uuid, Rules> {
             optional: false,
         },
         content: ContentRule::Templated {
-            exp_template_type: PROPOSAL_TEMPLATE_UUID_TYPE
-                .try_into()
-                .expect("Must be a valid UUID V4"),
+            exp_template_type: expect_uuidv4(PROPOSAL_TEMPLATE_UUID_TYPE),
+        },
+        parameters: ParametersRule::Specified {
+            exp_parameters_type: expect_uuidv4(PARAMETERS_TEMPLATE_UUID_TYPE),
```

Comment:
No no, it should reference to the same `CATEGORY_DOCUMENT_UUID_TYPE`
```suggestion
            exp_parameters_type: expect_uuidv4(CATEGORY_DOCUMENT_UUID_TYPE),
```